features/add_ressource_to_item.feature

```diff
@@ -0,0 +1,19 @@
+#language: fr
+
+Fonctionnalité: Ajouter une ressource en tant que fichier 
+
+
+Scénario: Ajouter une ressource en tant que fichier 
+
+Soit "AXN 009" l'item affiché
+Et l'utilisateur est connecté
+Quand l'utilisateur dépose "favicon.ico" comme ressource
+Alors la ressource "favicon.ico" est ajoutée
+
+
+Scénario: Consulter une ressource en tant que fichier 
```

Comment:
This is not really a different case of the feature. Please move it to the consultation of an item or just remove this scenario.